ReactUI/src/ui/components/materialUI/converter/AudioForm.jsx

```diff
@@ -0,0 +1,374 @@
+							className={classes.input}
+							id="contained-button-file"
+							multiple
+							type="file"
+							required
+						/>
+						<label htmlFor="contained-button-file">
+							<Button
+								variant="contained"
+								color="primary"
+								component="span"
+							>
+								Upload
+							</Button>
+						</label>
+					</CardContent>
+				</Card>
+			</Grid>
+			<Grid item xs={4}>
+				<Card className={classes.card}>
+					<CardHeader
+						className={classes.title}
+						title="Format"
+						titleTypographyProps={{ variant: "h6" }}
+					/>
+					<CardContent>
+						<Grid container spacing={6}>
+							<Grid item xs>
+								<FormControl
+									variant="outlined"
+									className={classes.formControl}
+									fullWidth
+								>
+									<InputLabel id="demo-simple-select-outlined-label">
+										Output Format
+									</InputLabel>
+									<Select
+										labelId="demo-simple-select-outlined-label"
+										id="demo-simple-select-outlined"
+										label="Output Format"
+										value={promp.outputFormat}
+										onChange={(e) =>
+											promp.setOutputFormat(
+												e.target.value
+											)
+										}
+										required
+									>
+										<MenuItem value="">
+											<em>-</em>
+										</MenuItem>
+										<MenuItem value={"Output1"}>
+											.wav
+										</MenuItem>
+										<MenuItem value={"Output2"}>
+											.mp2
+										</MenuItem>
+										<MenuItem value={"Output3"}>
+											.mp3
+										</MenuItem>
+                                        <MenuItem value={"Output4"}>
+											.mp4
+										</MenuItem>
+                                        <MenuItem value={"Output5"}>
+											.m4a
+										</MenuItem>
+                                        <MenuItem value={"Output6"}>
+											.flac
+										</MenuItem>
+                                        <MenuItem value={"Output7"}>
+											.ogg
+										</MenuItem>
+										<MenuItem value={"Output8"}>
+											.amr
+										</MenuItem>
+									</Select>
+								</FormControl>
+							</Grid>
+						</Grid>
+					</CardContent>
+				</Card>
+			</Grid>
+			<Grid item xs={4}>
+				<Card className={classes.card}>
+					<CardHeader
+						className={classes.title}
+						title="Bit Rate"
+						titleTypographyProps={{ variant: "h6" }}
+					/>
+					<CardContent>
+					<Grid container spacing={6}>
+							<Grid item xs>
+								<FormControl component="fieldset">
+									<FormControlLabel
+										control={
+											<Checkbox
+												checked={promp.constant}
+												onChange={(e) => {
+													promp.setConstant(
+														e.target.checked
+													);
+												}}
+												name="Constant"
+												color="primary"
+											/>
+										}
+										label="Constant"
+									/>
+										<Select
+											labelId="demo-simple-select-outlined-label"
+											id="demo-simple-select-outlined"
```

Comment:
This id is the same as the one in the line 79, 188, 233, 274 I think they all should have different ids